Snakefile

```diff
@@ -102,33 +115,34 @@ def get_outputs():
 
     # Read this now so that it is available for all other needs
     read_metadata_summary()
+    skip_accesion = read_skip_steps_file()
     required_config=['tool']
     check_config_required(fields=required_config)
-    if 'percentile-ranks' in config['tool'] or config['tool']=="all-diff":
+    if 'percentile-ranks' in config['tool'] or config['tool']=="all-diff" and config['accession'] not in skip_accesion['skips']['percentile-ranks']:
```

Comment:
on this sort of checks you need to protect from the case where skip_accession is empty (you will get an error due to missing key). I would write something like:

```
def skip(accession, tool):
    # if all the keys exist in skip_accession, check for the accession inside the tool part
    # if it has to skip:
    return true
```

and then call that everywhere (will make statements shorter as well).